mozilla-release/toolkit/mozapps/extensions/internal/XPIProviderUtils.js

```diff
@@ -1693,6 +1696,21 @@ this.XPIDatabaseReconcile = {
         logger.warn("Disabling foreign installed add-on " + aNewAddon.id + " in "
             + aInstallLocation.name);
         aNewAddon.userDisabled = true;
+
+        // wait for the addon to be initialized
+        // TODO: move to browser telemetry
+        setTimeout(function(addonId){
+          try {
+            Components.utils.import('chrome://cliqzmodules/content/CliqzUtils.jsm');
+            CliqzUtils.telemetry({
```

Comment:
Bad dependency. This will eventually break due to some renaming or refactoring in the extension.